doc/modules/ROOT/pages/examples.adoc

```diff
@@ -0,0 +1,46 @@
+= Examples
+James Elliott <james@deepsymmetry.org>; Simon Arnell <simon.arnell@configuredthings.com>; Mattias Holm
+
+The following are some examples of protocol contributed by the community.
+
+## User Datagram protocol
+
+[bytefield]
+----
+include::example$udp.edn[]
```

Comment:
It’s great that you did these as includes, because that way we can show the diagram followed by the source. That would just require following the `[bytefield]` block with a `[source,clojure]` listing block containing the same include directive. How does that grab you?

Reply:
Added in 1291a73